man/tpm2_policyauthorize.1.md

```diff
@@ -70,7 +70,7 @@ verification of the signature on the pcr policy digest using **tpm2_policyauthor
 ## Create a signing authority
 * openssl genrsa -out signing_key_private.pem 2048
 * openssl rsa -in signing_key_private.pem -out signing_key_public.pem -pubout
-* tpm2_loadexternal -G rsa -a n -u signing_key_public.pem -o signing_key.ctx\
+* tpm2_loadexternal -G rsa -a o -u signing_key_public.pem -o signing_key.ctx\
```

Comment:
Why does it matter if we don't have a verification ticket?

Reply:
The tpm2_policyauthorize command uses this verification ticket to verify that PCR digest is correctly signed by given public key in unsealing process.
e.g.
tpm2_policyauthorize -S session.ctx -o authorized.policy -f pcr.policy\
-n signing_key.name -t verification.tkt
![image](https://user-images.githubusercontent.com/22586787/46715834-7dfed680-cc16-11e8-83ad-86818ab3c33d.png)